addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -345,6 +346,11 @@ export class Wysiwyg extends Component {
             return savedVideo;
         };
 
+        const plugins = options.editorPlugins;
+        if (options.enableMoveNodePlugin) {
+            plugins.push(MoveNodePlugin);
```

Comment:
Why ? And what does it have to do with the PR ?

Reply:
It is explained at where we set the `enableMoveNodePlugin` parameter:
```
             // Since `MoveNodePlugin` requires wysiwyg's assets that are not
            // loaded at the time of the extractProps call, we use the following
            // property to instruct wysiwyg to add the `MoveNodePlugin`.
```